configuration/OVERVIEW.md

```diff
@@ -584,6 +585,83 @@ return result;
 Note that you can pass a large object to the form, which can then read any value, but doing so does noticeably slow the loading of the form. Because of this it is preferable to remove from the context any fields that are not being used. It is a good idea to future proof by maintaining the same structure so that fields can be added without needing to modify existing form calculations.
 ### Tips & Tricks <!-- TODO -->
 ### Troubleshooting <!-- TODO -->
+### Create A New Collect App
+
+#### Step 1: Download Android studio or Eclipse. 

```

Comment:
We should probably point to platform-agnostic installation instructions (so that we take care of WIN and *NIX users)